pages/quizzes.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+
+<html lang=" en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="author" content="Harshna Haswani">
+  <meta name="description" content="Test yourself and grow into a rockstar developer">
+  <title> RockStar Dev — All Quizzes</title>
+  <link rel="stylesheet" href="/styles/main.css">
+
+</head>
+
+<body class="quizzes ">
+
+
+  <nav class="nav nav-site" role="main">
+    <header>
+      <h1 class="heading"><a href="/pages/quizzes.html">RockStar Dev</a></h1>
+    </header>
+
+    <div class="form-item status-plain-outline">
+      <input type="search" name="search" id="search" class="form-input" placeholder="search" aria-label="search"
+        autocomplete="name">
+    </div>
+
+    <ul class="list list-inline">
+
+
+
+      <li class="list-item">
+        <a href="/pages/bookmark.html">bookmark</a>
+      </li>
+
+
+
+      <li class="list-item">
+        <a href="/pages/quizzes.html">All Quizzes</a>
+      </li>
+
+
+
+
+      <li class="list-item">
+        <a href="/pages/scoreboard.html">Your Scoreboard</a>
+      </li>
+
+
+    </ul>
+    <button class="btn bg-accent btn-logout"><a href="/">logout</a></button>
+  </nav>
+
+
+
+  <main class="main-content quizzes">
+    <aside class="filter center-x">
+      <form class="form" action="#">
+        <div class="form-item status-plain-outline">
+          <label for="range-rating">Quiz Rating</label>
+          <input type="range" min="1" max="5" value="3" class="form-input" id="range-rating">
+        </div>
+        <fieldset class="form-item status-plain-outline">
+          <legend>Category</legend>
+
+          <div>
+            <input type="checkbox" id="Database" name="category" value="Database">
+            <label for="Database">Database</label>
+          </div>
+          <div>
+            <input type="checkbox" id="JavaScript" name="category" value="JavaScript">
+            <label for="JavaScript">Vanilla JS</label>
+          </div>
+          <div>
+            <input type="checkbox" id="ReactJS" name="category" value="ReactJS">
+            <label for="ReactJS">React JS</label>
+          </div>
+          <div>
+            <input type="checkbox" id="NodeJS" name="category" value="NodeJS">
+            <label for="NodeJS">Node JS</label>
+          </div>
+          <div>
+            <input type="checkbox" id="HTML" name="category" value="HTML" checked>
+            <label for="HTML">HTML</label>
+          </div>
+          <div>
+            <input type="checkbox" id="CSS" name="category" value="CSS">
+            <label for="CSS">CSS</label>
+          </div>
```

Comment:
why the id in capital letters?

Reply:
Thanks for pointing it out, I'll change it